commerce_shipping.routing.yml

```diff
@@ -29,3 +29,28 @@ entity.commerce_shipment_type.collection:
     _title: 'Shipment types'
   requirements:
     _permission: 'administer commerce_shipment_type'
+
+entity.commerce_shipment.collection:
+  path: '/admin/commerce/orders/{commerce_order}/shipments'
+  defaults:
+    _entity_list: 'commerce_shipment'
+    _title: 'Shipments'
+  options:
+    parameters:
+      commerce_order:
+        type: 'entity:commerce_order'
+  requirements:
+    _permission: 'administer commerce_shipment'
+
+entity.commerce_shipment.edit_form:
+  path: '/admin/commerce/orders/{commerce_order}/shipment/{commerce_shipment}/edit'
+  defaults:
+    _entity_form: 'commerce_shipment.default'
+  options:
+    parameters:
+      commerce_order:
+        type: 'entity:commerce_order'
+      commerce_shipment:
+        type: 'entity:commerce_shipment'
+  requirements:
+    _commerce_shipment_operation_access: 'TRUE'
```

Comment:
**CHANGE REQUEST**

*Readability*

I believe the entities will be picked up automatically, at least when there is only one. I'd check removing the `options` section for both routes.

Also, why are these needed in the first place? If you are not missing any annotations on the Shipment entity the routes should be available automatically. See comments before. In that case these routes are not needed.